index.js

```diff
@@ -0,0 +1,66 @@
+const grid = document.querySelector('.grid');
+const palette = document.querySelector('.palette')
+let color;
+// const superPalette = document.querySelector('input');
+// console.log(superPalette)
+
+function createGrid(width, height) {
+  for (let i = 0; i < height; i++) {
+    let row = document.createElement('div');
+    row.classList.add('row');
+
+    for (let x = 0; x < width; x++) {
+      let pixel = document.createElement('div');
+      pixel.classList.add('pixel');
+      row.appendChild(pixel);
+    }
+    grid.appendChild(row);
+  }
+}
+
+createGrid(40,30);
+
+
+const colorsArray = ['red', 'orange', 'yellow', 'green', 'blue',
```

Comment:
This would have been better placed at the top. A convention is to have all variable definition located at the beginning of the file.